src/UbiHttp.cpp

```diff
@@ -134,21 +139,25 @@ double UbiHTTP::get(const char *device_label, const char *variable_label) {
     return ERROR_VALUE;
   }
 
-  uint16_t pathLength = _pathLength(device_label, variable_label);
-  char *path = (char *)malloc(sizeof(char) * pathLength + 1);
+  int pathLength = snprintf(NULL, 0, "/api/v1.6/devices/%s/%s/lv", device_label, variable_label);
```

Comment:
I would prefer to just replace this line where _pathLength is defined as it is already being reused, also the implementation already has a description.

Reply:
Following the principle of KISS (Keep it Simple, Studpid), read `snprintf` probably makes sense now, but giving semantic names to specific actions may reduce the time to fully understand the code as it is natural language.

Reply:
The change is reverted.